pinot-spi/src/main/java/org/apache/pinot/spi/config/table/TablePauseStatus.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.spi.config.table;
+
+import org.apache.pinot.spi.config.BaseJsonConfig;
+
+
+public class TablePauseStatus extends BaseJsonConfig {
+  private boolean _paused;
+  private ReasonCode _reasonCode;
+  private String _comment;
+  private long _timeInMillis;
```

Comment:
Can we change the time to human readable string, so that by just looking at ideal state we know when it was paused or resumed: `long _timeInMillis -> String _time`